tools/batchTestGenerator/cmd/root.go

```diff
@@ -0,0 +1,172 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+
+	batch "github.com/aws-observability/aws-otel-test-frameowrk/batchTestGenerator/internal"
+	"github.com/spf13/cobra"
+)
+
+type eksFields struct {
+	region      string
+	ampEndpoint string
+	clusterName string
+}
+
+type commandConfig struct {
+	runConfig     batch.RunConfig
+	rootCommand   cobra.Command
+	githubCommand cobra.Command
+	localCommand  cobra.Command
+	includeFlags  []string
+	eksARM64Flags eksFields
+	eksFlags      eksFields
+}
+
+func newCommandConfig() *commandConfig {
+	c := &commandConfig{
+		runConfig: batch.NewDefaultRunConfig(),
+	}
+
+	preRunSetup := func(cmd *cobra.Command, args []string) error {
+		if len(c.includeFlags) > 0 {
+			c.runConfig.IncludedServices = transformInclude(c.includeFlags)
+		}
+
+		if c.runConfig.TestCaseFilePath == "" {
+			//validate input/output set to cwd if necessary
+			defaultFileLoc, err := os.Getwd()
+			if err != nil {
+				return fmt.Errorf("failed to get cwd: %w", err)
+			}
+			c.runConfig.TestCaseFilePath = defaultFileLoc
+		}
+
+		eks64vars, err := transformEKSvars(c.eksARM64Flags)
+		if err != nil {
+			return fmt.Errorf("failed to transform eks arm 64 vars: %w", err)
+		}
+		c.runConfig.EksARM64Vars = eks64vars
+
+		eksVars, err := transformEKSvars(c.eksFlags)
+		if err != nil {
+			return fmt.Errorf("failed to transform eks vars: %w", err)
+		}
+		c.runConfig.EksVars = eksVars
+
+		if c.runConfig.MaxBatches < 1 {
+			return fmt.Errorf("max batches must be greater than 1")
```

Comment:
```suggestion
			return fmt.Errorf("max batches must be greater than 0")
```

Or fix the condition if it really must be <1.